### ISSUE TITLE
MountPoint format like df?

### ISSUE THREAD

**Issue by saluker** [deniszh/collectd-iostat-python#11]
Does the current plugin support MountPoint like df?  See https://collectd.org/wiki/index.php/Plugin:DF -- would be very handy if it doesn't.

**Comment by deniszh** [comment]
No, currently mountpoint is not supported - but it should be quite easy to add. Will try that.

**Comment by saluker** [comment]
Great.  Let me know if I can help testing.  I imagine the mountmount is actually going to need to be emitted now as a field.  Currently it is not, as far as I can tell.

**Comment by saluker** [comment]
Denis, need a hand with this?

**Comment by deniszh** [comment]
Sorry, was quite busy with working and personal stuff last week. Will try to find some time this weekend.

**Comment by saluker** [comment]
Thanks, Denis!  Look forward to trying it out.
